src/plugins/dashboard/public/application/listing/dashboard_listing.js

```diff
@@ -187,6 +188,19 @@ export class DashboardListing extends React.Component {
         dataType: 'string',
         sortable: true,
       },
+      {
+        field: `updated_at`,
+        name: i18n.translate('dashboard.listing.table.columnUpdatedAtName', {
+          defaultMessage: 'Updated at',
+        }),
+        dataType: 'date',
+        sortable: true,
+        description: i18n.translate('dashboard.listing.table.columnUpdatedAtDescription', {
+          defaultMessage: 'Updated at of the saved object',
+        }),
+        ['data-test-subj']: 'updated-at',
+        render: (updatedAt) => updatedAt && moment(updatedAt).format('MMM DD YYYY'),
```

Comment:
Wouldn't the time as well as the date be useful here?

Reply:
What about `Last updated` instead of `Updated Time` or `Updated at`?

Reply:
sure

Reply:
> Wouldn't the time as well as the date be useful here?

I will check with the product, for now, the update date column will be sufficient I guess. 
@razi284

Reply:
Ok, thanks a lot for the suggestion.
Utilized advanced settings and picked the right format:

<img width="1060" alt="Screen Shot 2022-02-14 at 10 29 18" src="https://user-images.githubusercontent.com/12782945/153836885-ec1ab884-f95f-4bff-819b-ada6a1b7a7e7.png">